tests/foundry/exclude

```diff
@@ -68,6 +68,9 @@ NonceTest.testNonce
 OwnerUpOnlyTest.testFailIncrementAsNotOwner
 OwnerUpOnlyTest.testIncrementAsNotOwner
 OwnerUpOnlyTest.testIncrementAsOwner
+PlainPrankTest.test_startPrankWithOrigin_true
+PlainPrankTest.test_stopPrank_true2
+PlainPrankTest.testFail_startPrank_false
```

Comment:
Hmmm... We should find a way to exercise this test and confirm that it fails instead of just disabling it.

Reply:
As far as I know, we haven't yet implemented the mechanisms for `testFail`, which I believe is the reason for the last test to be excluded. I will double-check that.